adapters/DTExchange/DTExchangeAdapter/GADMAdapterFyberUtils.m

```diff
@@ -72,7 +72,7 @@ GADVersionNumber GADMAdapterFyberVersionFromString(NSString *_Nonnull versionStr
     builder.spotID = spotID;
     builder.timeout = 10;
     builder.userData = extras.userData;
-    builder.muteAudio = extras.muteAudio;
+    IASDKCore.sharedInstance.muteAudio = extras.muteAudio;
```

Comment:
@2energycell 

Since this is not related to the builder, we can move it before building the request (i.e. line 69).

Also, can you confirm if the below check is needed? Publishers might not have set the `muteAudio`.
```
  if (extras.muteAudio) {
    IASDKCore.sharedInstance.muteAudio = extras.muteAudio;
  }
```

Reply:
correct; but I will submit one more PR shortly; it covers 2 previous PRs;

Reply:
@joshh-devrel this PR https://github.com/googleads/googleads-mobile-ios-mediation/pull/409
can be merged instead of 8.2.0 and 8.2.1 and it supports 8.2.0 SDK as well; thx;